Doc/source/release_notes.rst

```diff
@@ -42,6 +42,14 @@ New features
 now support reading from gzipped input files; filenames ending with
 ``.gz`` are assumed to be gzipped.
 
+`.datamodel` now has explicit support for operations using
+ISTP-compliant metadata; see `~.datamodel.dmarray` and
+`~.datamodel.SpaceData` for details. In particular note the plotting
+support with `~.datamodel.ISTPContainer.plot`.
+
+`~.plot.spectrogram.simpleSpectrogram()` supports plotting zeros as
+valid data.
```

Comment:
When log-scaling. If log-scaling is off zeros are always valid...

Reply:
Right.